engine/console/game.go

```diff
@@ -7,31 +7,47 @@ import (
 	"log"
 	"math/rand"
 	"os"
+	"strconv"
 	"time"
+
+	//"github.com/sanity-io/litter"
 )
 
 type Snake struct {
-	id    int          // snakep2p id
-	body  []core.Coord // coordinates of snakep2p's body
-	head  core.Coord   // coordinates of snakep2p's head
-	style tcell.Style  // snakep2p's style
+	alive bool         // whether snake is alive or not
+	Body  []core.Coord // coordinates of snakep2p's body
+	Head  core.Coord   // coordinates of snakep2p's head
+	Style tcell.Style  // snakep2p's style
 }
 
 type Game struct {
 	Ch             chan interface{} // communication channel
-	snakes         []Snake          // snakes' state: alive snakes with ID, head and body coordinates
-	food           []core.Coord     // food state: coordinates of food on the field
-	numAliveSnakes int              // number of alive snakes in the game
+	Snakes         map[int]*Snake   // snakes' state: alive snakes with ID, head and body coordinates
+	Food           map[int]core.Coord     // food state: coordinates of food on the field
+	NumAliveSnakes int              // number of alive snakes in the game
+	Over 	   	   bool				// whether game is over or not
```

Comment:
The comments can be easily inferred from the variable's name and type. Consider removing them, please.